packages/create-medusa-app/src/utils/get-current-os.ts

```diff
@@ -0,0 +1,10 @@
+export const getCurrentOs = (): string => {
+  switch (process.platform) {
```

Comment:
question: is this sufficient?

Reply:
Should be 😄 According to the Node.js docs it's the equivalent to [os.platform](https://nodejs.org/docs/latest-v16.x/api/os.html#osplatform) method. We can also ask someone with a different OS than mac to test it out.